app/presenters/update_password_presenter.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class UpdatePasswordPresenter
+  attr_reader :user, :required_password_change
```

Comment:
Similar to above, I'd expect references to this method to use the `?` suffix for predicate methods.

```suggestion
  attr_reader :user, :required_password_change

  alias_method :required_password_change?, :required_password_change

```